Fixes issue with machineAuth getSessionInfo for multiple audiences

rob5standingby opened the pull request:
## Why?
- Small bug with using `machineAuth` that I discovered while working on EMS API. When calling two functions from two different modules at the same time (for the first time), the incorrect token would get utilized for the second function call.

## What changed?
- Adjusted the function `getSessionInfo` to return a hash of promises, rather than a single promise

greg5green commented on src/sessionTypes/machineAuth.js:
```diff
@@ -101,8 +101,12 @@ class MachineAuth {
-    if (!this._tokenPromise) {
-      this._tokenPromise = axios
+    if (!this.tokenPromises) {
+      this.tokenPromises = {};
```
Maybe we should do this in the constructor?

rob5standingby replied:
Yeah good thinking.

rob5standingby commented:
@greg5green I've moved instantiation of `this.tokenPromises` to the constructor, and also renamed the var to `this._tokenPromises` to stick with the naming conventions. Let me know if this looks good!
